README.md

```diff
@@ -182,6 +182,16 @@ If you find PAWLS helpful for your research, please consider cite PAWLS.
 }
 ```
 
+### Troubleshooting
+
+1. Windows EOL format (CRLF) vs Linux (LF)
+
+Sometimes you may wondering why building docker image is not working and `ui` part is taking down. It's because some files were saved in incorrect format. To healing such files just go to the repo's root and paste following in terminal.
+
+```bash
+~ (cd ./ui && yarn && yarn lint:fix) # with brackets, to stay in same directory
+```
```

Comment:
Interesting.  The program is usually executed in a Docker container, so I wouldn't expect this to be an issue.

Are you trying to run the program on a Windows machine outside of Docker? Or are you still hitting this while using `docker-compose`?

Reply:
So, probably case was reproduced on older version of pawls. 

when we tried to `docker compose build` on windows 10, it thrown following issues.
![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/24553355/125269700-9e9d7300-e311-11eb-854a-38b280ecd66b.png)

Reply:
Interesting, I'm going to try to investigate a bit, just to make sure we don't inadvertently introduce a compatibility issue that impacts linux clients.

Thanks for your patience.

Reply:
Did the application actually fail to start in this scenario? My hope is that the application still starts, even though the linter is failing.

It's interesting that the linter in this case prefers the line-endings of the host OS, not of the container it's running in. That said I don't know a lot about running docker on Windows, as I don't use Windows :).

Reply:
How I might guess, all linter rules are configured to throw errors when code is not fit them and due to that fact building process interrupted.

Reply:
Ah, yup. And we execute a build before starting the application locally, even though we don't need to.

Thanks, that's super helpful. I'm going to make a quick tweak that should fix that and in doing so make the local development environment a bit faster to start. Thanks!

Reply:
This PR will at least allow the container to start when there's formatting errors;
https://github.com/allenai/pawls/pull/134

Thanks for catching this!